deeprank2/tools/target.py

```diff
@@ -83,22 +83,40 @@ def add_target(graph_path: Union[str, List[str]], target_name: str, target_list:
             print(f"no graph for {hdf5}")
 
 
-def compute_targets(pdb_path: str, reference_pdb_path: str) -> Dict[str, Union[float, int]]:
+def compute_ppi_scores(pdb_path: str, reference_pdb_path: str) -> Dict[str, Union[float, int]]:
 
     """
-    Compute targets and outputs them as a dictionary.
-    For classification:
-       - binary (scalar value is expected to be either 0 or 1)
-       - capri_classes (scalar integer values are expected)
+    Compute structure similarity scores and return them as a dictionary.
+    Such measures have been developed for evaluating the quality of the PPI models produced by
```

Comment:
the first sentence of the docstring should be on the same line as the triple-quote and then an empty line before the longer explanation.

Also, I reworded it slightly (can't do a code suggestion for some reason:

```
    """Compute structure similarity scores and return them as a dictionary.
    
    These measures have been developed for evaluating the quality of the PPI models produced by
    computational methods (docking models), and all of them compare the structural similarity
    between the decoys (computationally generated structures) and the experimentally solved native
    structures. To calculate these measures, the interface between the two interacting protein molecules
    is defined as any pair of heavy atoms from the two molecules within 5Å of each other.

    - lmrsd (ligand root mean square deviation) is a float value calculated for the backbone of
        the shorter chain (ligand) of the model after superposition of the longer chain (receptor).
        Lower scores represent better matching than higher scores.
    - imrsd (interface rmsd) is a float value calculated for the backbone atoms of the interface residues 
        (atomic contact cutoff of 10Å) after superposition of their equivalents in the predicted complex (model)
        Lower scores represent better matching than higher scores.
    - fnat (fraction of native contacts) is the fraction of native interfacial contacts preserved in
        the interface of the predicted complex. The score is a float in the range [0, 1], where higher values
        respresent higher quality.
    - dockq (docking model quality) is a continuous quality measure for docking models that instead of classifying into different
        quality groups. It combines fnat, lmrs, and irms and yields a float score in the range [0, 1], where
        higher values respresent higher quality.    
    - binary (bool): True if the irmsd is lower than 4.0, meaning that the decoy is considered high quality
        docking model, otherwise False.
    - capri classification (int). The possible values are: 1 (high quality, irmsd < 1.0), 2 (medium, irmsd < 2.0), 
        3 (acceptable, irms < 4.0), 4 (incorrect, irmsd >= 4.0)
    See https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.10393 for more details about capri_classes, lrmsd, irmsd, and fnat.
    See https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0161879 for more details about dockq.

    Args:
        pdb_path (str): Path to the decoy.
        reference_pdb_path (str): Path to the reference (native) structure.

    Returns: a dictionary containing values for lrmsd, irmsd, fnat, dockq, binary, capri_class.
    """
```